graph/telemetry/istio/appender/appender.go

```diff
@@ -34,14 +34,14 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 				requestedAppenders[IdleNodeAppenderName] = true
 			case IstioAppenderName:
 				requestedAppenders[IstioAppenderName] = true
+			case MeshCheckAppenderName:
```

Comment:
If we change the appender name and don't add  backward-compatible test for "sidecarsCheck" we need to make sure this backend will only be called by compatible clients.  I'm not 100% sure if we could have an older OSSMC calling this backend, probably not, but if we have any doubt we could add the additional check.

Reply:
It makes sense to just in case be backwards compatible. I have included the sidecarsCheck.